core/src/main/scala/org/apache/spark/util/taskListeners.scala

```diff
@@ -55,14 +55,16 @@ class TaskCompletionListenerException(
   extends RuntimeException {
 
   override def getMessage: String = {
-    if (errorMessages.size == 1) {
```

Comment:
Good catch !
Btw, file name should probably have been `taskListeners.scala` (unrelated to this PR) ?